src/overlays/actors/ovl_En_Ma1/z_en_ma1.c

```diff
@@ -437,14 +433,24 @@ s32 EnMa1_OverrideLimbDraw(PlayState* play, s32 limbIndex, Gfx** dList, Vec3f* p
 
 void EnMa1_PostLimbDraw(PlayState* play, s32 limbIndex, Gfx** dList, Vec3s* rot, void* thisx) {
     EnMa1* this = (EnMa1*)thisx;
-    Vec3f vec = D_80AA16B8;
+    Vec3f vec = { 800.0f, 0.0f, 0.0f };
```

Comment:
`offset` would be a slightly more descriptive name than `vec`, while you're here